literature.bib

```diff
@@ -2198,6 +2198,97 @@ @inproceedings{FMS+:SANER17
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2016 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@phdthesis{K16,
+	title={{The Role of Complex Constraints in Feature Modeling}},
+	author={Kn{\"u}ppel, Alexander},
+	school = {TU Braunschweig},
+	address = {Germany},
+	year=2016
+}
+
+@article{KR:QREI16,
+	title={{An Optimal Solution for Test Case Generation Using ROBDD Graph and PSO Algorithm}},
+	author={Kalaee, Akram and Rafe, Vahid},
+	journal=QREI,
+	volume={32},
+	number={7},
+	pages={2263--2279},
+	year=2016,
+	publisher=Wiley,
+	address = WileyMonterey
+}
+
+@article{TS:JEA16,
+	author    = {Takahisa Toda and Takehide Soh},
+	title     = {{Implementing Efficient All Solutions SAT Solvers}},
+	journal   = JEA,
+	volume    = {21},
+	number    = {1},
+	month     = NOV,
+	pages     = {1.12:1--1.12:44},
+	doi       = {10.1145/2975585},
+	publisher = ACM,
+	address   = NY,
+	year      = 2016,
+}
```

Comment:
It is not a big deal, but before I always tried to have spacing consistent:

"    key = value,"

Reply:
Normalized all spaces with `sed -e "s/\s*=\s*/ = /g"`